junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/MethodArgumentsProvider.java

```diff
@@ -37,8 +38,10 @@ public void accept(MethodSource annotation) {
 		Object testInstance = context.getTestInstance().orElse(null);
 		// @formatter:off
 		return Arrays.stream(methodNames)
+				.filter(Objects::nonNull)
+				.map(methodName -> methodName.isEmpty() && context.getTestMethod().isPresent() ? context.getRequiredTestMethod().getName() : methodName)
```

Comment:
Under what circumstances would the test method not be _present_?

Reply:
Honestly, I don't know - as I'm not very deep in the internals I prefer defensive programming so only my particular requirement is covered to minize the probability of side effects.

Reply:
Well, I don't think the test method could ever possibly be absent, but we can polish that after we merge the PR if we deem necessary.

@marcphilipp, thoughts?